projects/cian-lst/index.js

```diff
@@ -3,7 +3,7 @@ const config = {
     "0xcDd374F491fBF3f4FcF6E9023c99043774005137",
     "0xB8c0c50D255B93f5276549cbA7F4bf78751A5D34",
     "0x88508306E43FCe43F7f2c1e7D73c88cf6a523f6C",
-    "0xD34f59E172cF3915f56C96A3037Ac554A7399D77",
+    // "0xD34f59E172cF3915f56C96A3037Ac554A7399D77", // PYUSD Optimized Long-Short (variant 1)
```

Comment:
I would instead do permitFailure on line 22, imo, it is equally bad (or worse) solution but at least we will get the tvl of the vault again if it does work again